src/pages/ServiceDetails/ServiceInfo/ServiceInfoDescription.tsx

```diff
@@ -57,9 +58,15 @@ class ServiceInfoDescription extends React.Component<ServiceInfoDescriptionProps
               <div>
                 <strong>Type</strong> {this.props.type ? this.props.type : ''}
               </div>
-              <div>
-                <strong>IP</strong> {this.props.ip ? this.props.ip : ''}
-              </div>
+              {this.props.type !== 'ExternalName' ? (
```

Comment:
Not a blocker but there is an effort underway to remove string literals. 'type' would be a good candidate for stonger typing.

Reply:
Totally agree @jshaughn! Already changed that :)
Thanks for the review!